lib/tentacat.ex

```diff
@@ -121,6 +121,8 @@ defmodule Tentacat do
   def request_with_pagination(method, url, auth, body \\ "") do
     resp = request!(method, url, JSX.encode!(body), authorization_header(auth, extra_headers() ++ @user_agent), extra_options())
     case process_response(resp) do
+        {status, redirect_response} when status in [301, 302, 307] -> redirect_response
+        request_with_pagination(method, location_header(resp), auth)
```

Comment:
something is weird with the indentation here @micheldoumit and also there is this warning now during compilation https://travis-ci.org/edgurgel/tentacat/jobs/273683374#L710